config.py

```diff
@@ -87,6 +92,11 @@ class TestingConfig(Config):
     WTF_CSRF_ENABLED = False
 
 
+    @classmethod
```

Comment:
Hey can you just explain what the cls argument is to init app?

Reply:
Of course. These two comments are related so I'll answer both here. 

It's a compatibility issue to do with the 'config[config_name].init_app(app)' call from line 33 in app/__init__.py. There are two choices, follow the base class syntax (class Config) and use @staticmethod with only one input i.e def init_app(app):, or follow the super class syntax (e.g. class ProductionConfig) and have two inputs, the instance itself and app i.e. def init_app(cls, app):.

But the approach can't be mixed or one would break the other syntax.

I went with the super class syntax to match with ProductionConfig as they are kinda an equivalent level of class, compared to base Config at least!